src/screens/Ticketing/Purchase/Travellers/index.tsx

```diff
@@ -35,14 +36,25 @@ const Travellers: React.FC<TravellersProps> = ({
 
   const {userProfilesWithCount, addCount, removeCount} = useUserCountState();
 
+  const {tariffZones} = useTicketState();
+  const {
+    fromTariffZone = tariffZones[0],
+    toTariffZone = tariffZones[0],
```

Comment:
For now just using tariffZones[0] as the default, which conveniently is Sone A. We probably want a more reliable method for selecting the default zone. Should the default zone already be flagged when it is returned from backend?

Reply:
When trying this locally now, my default zone was "Sone B3". This should probably be part of the "data washing" which happens in Remote Config.

Reply:
> Seems like there's some problems with the text input? Unsure if this is just me. I have to type letters several times before they register

Ahh. I've used `debouncedText` instead of `text` as the InputField value. Fixing.

Reply:
Yes. The ordering of the tariff zones from the reference data endpoint has changed now. 

We can probably change this later, this is a "problem" we have with all the reference data now. More ticket types than only "Enkeltbillett" are shown, user profiles are not ordered, and no (good) way to specify default tariff zone. This will be improved when it is fully decided where to put the "data washing".